terraform/batchCache/main.tf

```diff
@@ -0,0 +1,31 @@
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.8.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.region
+}
+
+resource "aws_dynamodb_table" "batch_successful_cache" {
+  name         = "BatchTestCache${var.bucketUUID}"
```

Comment:
I don't know that a UUID is needed here since DDB tables are not global.

Reply:
[AWS docs](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_CreateTable.html) state DDB table names must be regionally unique